src/re_frame/interceptor.cljc

```diff
@@ -1,7 +1,9 @@
 (ns re-frame.interceptor
   (:require
-    [re-frame.loggers :refer [console]]
-    [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]))
+   [re-frame.loggers :refer [console]]
+   [re-frame.interop :refer [ratom? empty-queue debug-enabled?]]
+   [clojure.set :as set]
+   ))
```

Comment:
Excuse me, I am not quite sure about what you mean. Could you tell me the right way so that I can fix it. Thanks.